external-resolver/src/main/scala/org/genivi/sota/resolver/db/DeviceRepository.scala

```diff
@@ -90,82 +92,75 @@ object DeviceRepository {
    */
 
   // scalastyle:off
-  class InstalledPackageTable(tag: Tag) extends Table[(Namespace, Uuid, PackageId)](tag, "InstalledPackage") {
-    def device         = column[Uuid]("device_uuid")
-    def namespace      = column[Namespace]("namespace")
-    def packageName    = column[PackageId.Name]("packageName")
-    def packageVersion = column[PackageId.Version]("packageVersion")
+  class InstalledPackageTable(tag: Tag) extends Table[(Uuid, UUID)](tag, "InstalledPackage") {
+    def device = column[Uuid]("device_uuid")
+    def packageUuid = column[UUID]("package_uuid")
```

Comment:
Having all those different alias and types feels like overkill sometimes so I am not sure about that. Anyway we should definitely unify all these whatever we choose.